scripts/end_to_end_test/end_to_end_test.yaml

```diff
@@ -239,3 +239,26 @@ steps:
   - sleep: 5
   - run_python_test: { "script": ".smart_contract_tests.malleability_test" }
 ...
+
+---
+test_description: >
+  DISABLED - Test synergetic contract fees
+
+setup_conditions:
```

Comment:
Would also be good it we can think of a way to catch the negative case here in and end 2 end test. I.e. when the work is not completed because there are no funds available